lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebar.test.tsx

```diff
@@ -5,98 +5,105 @@
+
+    await runTimers();
+
+    const toggleBtn = view.getByRole('button', { name: /collapse sidebar/i });
+
+    expect(toggleBtn).toHaveAccessibleName('Collapse Sidebar');
+    expect(toggleBtn).toHaveAttribute('aria-expanded', 'true');
+    expect(toggleBtn).toHaveAttribute('aria-controls', sidebarId);
+
+    await user.hover(toggleBtn);
+
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toBeInTheDocument();
+    expect(tooltip).toHaveTextContent('Collapse Sidebar');
   });
 
-  it('renders a button with toggleCloseIcon if isOpen is false', function() {
-    const component = getShallowComponent({ isOpen: false }),
-        toggleBtn = component.find(NxButton),
-        sidebarId = component.prop('id');
-
-    expect(toggleBtn).toMatchSelector('.nx-global-sidebar__toggle');
-    expect(toggleBtn).toHaveProp('variant', 'icon-only');
-    expect(toggleBtn).toHaveProp('title', 'Expand Sidebar');
-    expect(toggleBtn).toHaveProp('aria-expanded', false);
-    expect(toggleBtn).toHaveProp('aria-controls', sidebarId);
-    expect(toggleBtn.find(NxFontAwesomeIcon)).toHaveProp('icon', faBiohazard);
+  it('renders an icon-only button with tooltip "Expand Sidebar" if isOpen is false', async function() {
+    const view = quickRender({ isOpen: false }),
+        sidebarId = view.container.firstElementChild?.getAttribute('id'),
+        user = userEvent.setup();
+
+    await runTimers();
+
+    const toggleBtn = view.getByRole('button', { name: /expand sidebar/i });
+
+    expect(toggleBtn).toHaveAccessibleName('Expand Sidebar');
+    expect(toggleBtn).toHaveAttribute('aria-expanded', 'false');
+    expect(toggleBtn).toHaveAttribute('aria-controls', sidebarId);
+
+    await user.hover(toggleBtn);
+
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toBeInTheDocument();
+    expect(tooltip).toHaveTextContent('Expand Sidebar');
   });
 
   it('renders passed in children', function() {
     const children = (
       <>
-        <div className="child-1">
-          <p>I am children</p>
+        <div className="child-1" data-testid="div1">
+          <p data-testid="p1">I am children</p>
         </div>
-        <div className="child-2"></div>
+        <div className="child-2" data-testid="div2"></div>
       </>
     );
-    const component = getShallowComponent({ children });
-    expect(component.find('.child-1')).toExist();
-    expect(component.find('.child-2')).toExist();
+    const view = quickRender({ children });
+    expect(view.getByTestId('div1')).toBeInTheDocument();
+    expect(view.getByTestId('p1')).toBeInTheDocument();
+    expect(view.getByTestId('div2')).toBeInTheDocument();
   });
 
-  it('calls onToggleClick when toggle button is pressed', function() {
-    const component = getShallowComponent(),
-        btn = component.find(NxButton);
+  it('calls onToggleClick when toggle button is pressed', async function() {
+    const onToggleClick = jest.fn(() => {}),
```

Comment:
Comparing tests to the docs, there is no documentation for `onToggleClick` in the gallery docs (even though it's listed as a prop in `types` and in all the examples). Probably a separate ticket, but I'm assuming we need to add it in

Reply:
> Comparing tests to the docs, there is no documentation for `onToggleClick` in the gallery docs (even though it's listed as a prop in `types` and in all the examples). Probably a separate ticket (?), but I'm assuming we need to add it in

Yep, I see that it's missing. Created a ticket for it https://issues.sonatype.org/browse/RSC-1449